rct229/data_fns/table_3_2_fns.py

```diff
@@ -0,0 +1,64 @@
+import rct229
+from rct229.data import data
+from rct229.data_fns.table_utils import find_osstd_table_entry
+
+# This dictionary maps the ClimateZone2019ASHRAE901 enumerations to
+# the corresponding climate zone values in the OSSTD file
+# ashrae_90_1_table_3_2.json
+"""
+The enumeration has been updated to match the following climate zones from the standard.
+CZ0A, CZ0B -> "0"
+CZ1A, CZ1B -> "1"
+CZ2A, CZ2B -> "2"
+CZ3A, CZ3B -> "3A,3B"
+CZ3C -> "3C"
+CZ4A, CZ4B -> "4A,4B"
+CZ4C -> "4C"
+CZ5A, CZ5B, CZ5C -> "5"
+CZ6A, CZ6B -> "6"
+CZ7 -> "7"
+CZ8 -> "8"
+
+"""
+climate_zone_enumeration_to_climate_zone_type_map = {
```

Comment:
Since the column in Table 3.2 is called "Climate Zone" and you have rightly used the field name "climate_zone" in the json data file, I believe the name for this map should be `climate_zone_enumeration_to_climate_zone_map` (I've dropped the word "type").